src/sass/_tnrisDownloadTemplateDownload.scss

```diff
@@ -111,4 +74,8 @@ styles for component: TnrisDownloadTemplateDownload
     padding: 35px;
   }
 
+  .template-content-div-subheader {
+    color: $nest-text;
+  }
```

Comment:
can you add `padding-bottom: 8px;` here to give a little room between the sub-header and the top of the map?

Reply:
yep, good idea. i'll do that now.

Reply:
i added 8px to top and bottom because on coverage map there is text at the bottom and it could use that padding as well. let me know if this works.

Reply:
Looks good!